frontend/src/components/auth/components/sign-in-form/sign-in-form.tsx

```diff
@@ -48,7 +48,9 @@ const SignInForm: FC<Props> = ({ onSubmit }) => {
           />
         </div>
         <div className={styles.buttonWrapper}>
-          <Button type="submit" label="Sign In" btnColor="blue" />
+          <div>
+            <Button type="submit" label="Sign In" btnColor="blue" />
+          </div>
```

Comment:
We don't need 2 wrappers for a button, use same approach in other places too
```suggestion
          <Button type="submit" label="Sign In" btnColor="blue" />
```
```
.buttonWrapper {
  max-width: 215px;
  margin: 0 auto;
}
```

Reply:
@Liza-Veis I have tried that, it didn't work :smiling_face_with_tear: 
The button takes the provided 215px width; We can make it work with max-width: 93px; but in that case we'll be setting min-width: 93px for a button and max-width: 93px for a wrapper. It's the same think as setting width: 93px, which we don't want to do

![image](https://user-images.githubusercontent.com/61088675/189484739-8627d958-5901-4064-a957-fd1f81f33a1a.png)

![image](https://user-images.githubusercontent.com/61088675/189484767-3651a67b-8d19-4aa1-a859-7070ef4aced2.png)

![image](https://user-images.githubusercontent.com/61088675/189484784-b9e61232-9ed4-49c9-b627-349d0bb146c0.png)